### ISSUE TITLE
Case sensitivity

### ISSUE THREAD

**Issue by forthrin** [dumblob/mysql2sqlite#87]
I see you don't have time to maintain this excellent tool anymore, but anyway here's one more for the heap.

It seems that MySQL is [case insensitive by default](http://dev.mysql.com/doc/refman/5.0/en/case-sensitivity.html), while SQLite is case sensitive by default.

Thus, when connecting to the newly created SQLite DB, INSERT IGNORE statements will yield different results when handling duplicates on UNIQUE columns compared to when using the original MySQL DB.

**Comment by dumblob** [comment]
You're right my time is very limited and I prefer to rather not maintain it than to accept badly reviewed PRs. I should probably write about this in the readme at some point...

Still feel free to make suggestions and ideally a PR how to go about the case sensitivity issue (I didn't check whether it appears also when using the required dump options as described in readme).
